File: openapi-client/php-dt/src/App/DTO/RawMail.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Raw Email Object
 */
class RawMail
{
    /**
     * The entire email contents
     */
    #[DTA\Data(field: "raw_email")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $raw_email = null;

}

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/RawMail.scala

package org.openapitools.client.model


case class RawMail (
    /* The entire email contents */
    _rawEmail: String
)
object RawMail {
    def toStringBody(var_rawEmail: Object) =
        s"""
        | {
        | "rawEmail":$var_rawEmail
        | }
        """.stripMargin
}
